guides/source/debugging_rails_applications.md

```diff
@@ -337,7 +336,6 @@ Processing by ArticlesController#index as HTML
    10:     respond_to do |format|
    11:       format.html # index.html.erb
    12:       format.json { render json: @articles }
-
 (byebug)
```

Comment:
Yep, and you seem to keep it updated (56ed162bf37a505707455055e24ae40d98d9741b). Updating the software versions used in this guide is the whole point of this PR, that's why I rerun all commands.